notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.    """
</pre><p>What is this commented out code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
I think it's an example of how to define a custom transform. It shows how the custom transform function `plus_one_transform` was defined for `transform_config_with_custom_transform`. A user can uncomment that code and modify it create their own customer transform, so an example is helpful.